docs/java-application-development/kura-workspace-setup.md

```diff
@@ -0,0 +1,152 @@
+# Kura Workspace Setup
+
+This document describes how to set up the Kura workspace development environment, which consists of the following components:
+
+- JVM (Java JDK SE 8)
+- Eclipse IDE
+- Kura Workspace
+
+**This setup will allow you to develop applications or bundles running on Kura. It will install only the APIs and the examples.** If you want to contribute to the Eclipse Kura project follow [this guide](./development-environment-setup.md) instead.
```

Comment:
```suggestion
**This setup will allow you to develop applications or bundles running on Eclipse Kura. It will install only the APIs and the examples.** If you want to contribute to the Eclipse Kura project follow [this guide](./development-environment-setup.md) instead.
```